.travis.yml

```diff
@@ -29,6 +61,7 @@ matrix:
       install:
         - export CC='ccache gcc-6' CXX='ccache g++-6' JOBS=2
         - ./configure
-        - make -j2 V=
+        # We already have a compile log in the above job
+        - make -j2 > /dev/null
       script:
-        - PARALLEL_ARGS='--flaky-tests=skip' make -j1 test
+        - TEST_CI_ARGS="--flaky-tests=skip --report default" make -j1 V= test-ci
```

Comment:
`test-ci` is https://github.com/nodejs/node/blob/61ef9df958f98bed5159d9d4553ea5b2f894272f/Makefile#L502-L512

where
https://github.com/nodejs/node/blob/61ef9df958f98bed5159d9d4553ea5b2f894272f/Makefile#L473

so specifying `default` in `TEST_CI_ARGS` is redundant.

Reply:
From the Travis run:

https://travis-ci.com/nodejs/node/jobs/188991243#L4136-L4138
```console
/usr/bin/python2.7 tools/test.py -j 2 -p tap --logfile test.tap \
	--mode=release --flaky-tests=run \
	--flaky-tests=skip --report default default addons js-native-api node-api doctool
```

Looking again we can probably set `FLAKY_TESTS=skip` if using `test-ci`.

Reply:
Ack